t/24-worker-webui-connection.t

```diff
@@ -455,15 +493,61 @@ qr/Ignoring WS message from http:\/\/test-host with type livelog_stop and job ID
                     type => 'grab_job',
                     job  => {id => 'but no settings'}});
         },
-        qr/Refusing to grab job.*because the provided job is invalid.*/,
+        qr/Refusing to grab job.*: the provided job is invalid.*/,
         'ignoring grab job if no valid job info provided',
     );
+    combined_like(
+        sub {
+            $command_handler->handle_command(
+                undef,
+                {
+                    type     => 'grab_jobs',
+                    job_info => {sequence => ['foo']},
+                });
+        },
+        qr/Refusing to grab jobs.*: the provided job info lacks job data or execution sequence.*/,
+        'ignoring grab multiple jobs if job data',
+    );
+    combined_like(
+        sub {
+            $command_handler->handle_command(
+                undef,
+                {
+                    type     => 'grab_jobs',
+                    job_info => {sequence => 'not an array', data => {42 => 'foo'}},
+                });
+        },
```

Comment:
just regarding syntax, this looks pretty verbose. Maybe that can be extracted into a function?

Reply:
It's verbose, but also well structured.